Extension/package.json

```diff
@@ -1431,7 +1431,7 @@
     "tslint-no-unused-expression-chai": "0.0.3",
     "typescript": "^2.9.2",
     "vrsource-tslint-rules": "^5.8.3",
-    "vscode": "^1.1.30"
+    "vscode": "^1.1.33"
```

Comment:
Why are we incrementing this?

Reply:
It's a bug fix update from running npm install.

Reply:
As long as it doesn't update our min version dependency, then this is fine.